### PR TITLE
Document Botometer reliability assessment

### PR DESCRIPTION
**Pull request by MattiSG**

### CODE REVIEW

File: _pages/encyclopedia/2-detection/Social-bots.md [hunk 1]
```diff
@@ -52,6 +52,43 @@ In order to use Botometer, one needs to authentify oneself. Indeed, it heavily r
...
 
+* **Reliability assessment**
+
+Botometer being closed source and having been built in 2016 from a US-based dataset, the French Office of the Ambassador for Digital Affairs independently audited the reliability of Botometer for non-American accounts in March 2019.
+
+##### Methodology
+
+Between the 26th and 28th February 2019, manual annotations were made by 12 French students of [Sciences-Po Saint-Germain-en-Laye](https://www.sciencespo-saintgermainenlaye.fr) (M1 level). They were presented with an initial 20 Twitter accounts representing authentic and inauthentic accounts, asked to classify them as pairs of certainty adverb and authentic / inauthentic types (e.g. “certainly a bot”, “probably a human”…).
 Each annotation was made by pairs of students and then reviewed by another, independent pair.
 Feedback was given by experts for the initial 20, and then on demand.
```
Comment (BerylBogui) [inline _pages/encyclopedia/2-detection/Social-bots.md]: "an initial file of 20" ?

File: _pages/encyclopedia/2-detection/Social-bots.md [hunk 2]
```diff
@@ -52,6 +52,43 @@ In order to use Botometer, one needs to authentify oneself. Indeed, it heavily r
...
+
+Students were then asked to select and classify another hundred account, with no specific criteria for selection. An exploratory algorithm was then designed and implemented, selecting accounts that were likely to be bots according to the most successful criteria as spotted by the human classifiers. A subset (312) of these accounts were then manually classified, once again by two independent pairs of students. A random subset (~ 80) of these annotations was then reviewed another time by disinformation experts. More information on methodology and results can be found on [sismo.quaidorsay.fr](http://sismo.quaidorsay.fr).
+
+Annotations were then transformed into numbers for easier comparison with Botometer scores, according to the following table:
+
+Manual classification        | Estimated Botometer score range
+-----------------------------|--------------------------------
+sans aucun doute artificiel  | 4.2–5.0 score
```
Comment (BerylBogui) [inline _pages/encyclopedia/2-detection/Social-bots.md]: Traduire les types

File: _pages/encyclopedia/2-detection/Social-bots.md [hunk 3]
```diff
@@ -52,6 +52,43 @@ In order to use Botometer, one needs to authentify oneself. Indeed, it heavily r
...
+certainement artificiel      | 3.8–4.2 score
+probablement artificiel      | 3.2–3.8 score
+peut-être *                  | 2.0–3.2 score
+probablement authentique     | 1.4–2.0 score
+certainement authentique     | 1.0–1.4 score
+sans aucun doute authentique | 0.0–1.0 score
+
+
Botometer scores were then calculated on the classified accounts on 19th and 24th Apr 2019.
```
Comment (BerylBogui) [inline _pages/encyclopedia/2-detection/Social-bots.md]: > scores of the classified accounts were then calculated on the 19th ...

File: _pages/encyclopedia/2-detection/Social-bots.md [hunk 4]
```diff
@@ -52,6 +52,43 @@ In order to use Botometer, one needs to authentify oneself. Indeed, it heavily r
...
+This time difference in computation is a human error and problematic, as some accounts were not available anymore and differences in assessment can also be explained by the time gap (a seemingly inauthentic account in February might become more human-like two months later, and the other way around).
+
+The difference is thus necessarily overestimated.
+
+
+##### Results
+
+![Botometer assessment](/assets/img/botometer-validation.png)
+
+Botometer seems to have more certainty than our human classifiers, however it also had [seven more weeks](#limitations) worth of data to assess the accounts. Due to this time difference in assessment, we cannot conclude as to the global reliability of Botometer. We can however see that the classification of “obviously inauthentic” accounts seems to be as properly made by Botometer as it is by humans.
```
Comment (BerylBogui) [inline _pages/encyclopedia/2-detection/Social-bots.md]: > conclude on

Comment (BerylBogui) [inline _pages/encyclopedia/2-detection/Social-bots.md]: > "accurately" plutôt que "properly"?

**Comment by MattiSG** [comment]
Thanks @BerylBogui for the corrections ❤️ 😃

**Pull request merged**